nfs-ganesha/build/build_deb

```diff
@@ -3,14 +3,21 @@ set -ex
 
 # Only do actual work when we are a DEB distro
 if test "$DISTRO" != "debian" -a "$DISTRO" != "ubuntu"; then
-    exit 0
+    exit 1
 fi
 
+REPO_URL="https://shaman.ceph.com/api/repos/ceph/$CEPH_BRANCH/$CEPH_SHA1/$DISTRO/$DIST/repo"
+if `curl --fail -L $REPO_URL`; then
+  echo "Ceph debian lib repo exists in shaman"
+else
+  echo "Ceph debian lib repo does NOT exist in shaman"
+  exit 1
+fi
 
 # We need this for system, to build ganesha, and to enable FSAL_CEPH and 
 # FSAL_RGW in the .spec file when cmake command runs
```

Comment:
It's fine to use the `-d` flag when running `dpkg-buildpackage` *with* `-S`. In fact that is how the `git-buildpackage` tool does it.

It will save time in the job if you don't have to install all these dependency packages here, plus you'll avoid the maintenance burden of trying to keep this list in sync with whatever the nfs-ganesha `/debian/control` file has for `Build-Depends`.